### PR TITLE
Index Page Content

### PR DESCRIPTION
**Pull request by stefankraus-wf**
Replaced filler content with relevant content on the index page. 

DOES NOT INCLUDE 

images
layout changes
Link changes
Blog section preview

### CODE REVIEW

File: src/pages/index.md [hunk 1]
```diff
@@ -1,70 +1,26 @@
...
+subheading: Education and opportunities for underserved youth
 mainpitch:
-  title: Why Kaldi
+  title: Why Pi515
   description: >
-    Kaldi is the coffee store for everyone who believes that great coffee
-    shouldn't just taste good, it should do good too. We source all of our beans
-    directly from small scale sustainable farmers and make sure part of the
-    profits are reinvested in their communities.
+    We go further than just the technical skills that our students deserve. Our students receive supplementary skills in College Prep, Financial Literacy, and Entrepreneurship. These vital skills are taught by our program to amplify our impact to maximize success.
```
Comment (nickolaswilson-wf) [inline src/pages/index.md]: >College Prep, Financial Literacy, and Entrepreneurship

Are these supposed to be capitalized?

File: src/pages/index.md [hunk 2]
```diff
@@ -1,70 +1,26 @@
...
 description: >-
-  Kaldi is the ultimate spot for coffee lovers who want to learn about their
-  java’s origin and support the farmers that grew it. We take coffee production,
-  roasting and brewing seriously and we’re glad to pass that knowledge to
-  anyone.
+  Giving opportunities that inspire young people to explore, discover, and create. By using the talent that we see in underrepresented students, we are able to make a real impact in their lives and watch as they go on to shape their communities. By starting at the source, we can organically grow a culture of ambition and success. The triumphs of our students goes on to be reflected in making the technology world more diverse and bringing the voices that are so often unheard to the forefront.
```
Comment (nickolaswilson-wf) [inline src/pages/index.md]: >The triumphs of our students goes
-> 
>The triumphs of our students go

**Review by ericcheatham — APPROVED** [review]

File: src/pages/index.md [hunk 3]
```diff
@@ -1,70 +1,26 @@
...
 intro:
   blurbs:
     - image: /img/coffee.png
       text: >
-        We sell green and roasted coffee beans that are sourced directly from
-        independent farmers and farm cooperatives. We’re proud to offer a
-        variety of coffee beans grown with great care for the environment and
-        local communities. Check our post or contact us directly for current
-        availability.
+        Pi515 successfully creates a safe space where students from a wide array of backgrounds, socio-economic and social status, and nationalities come together to learn from our program and each other. Individually, they learn to build skills and self-confidence. Together, they learn to build community – and the community is part of what keeps them invested in being part of the program. We provide the program at no cost to their families and give them a laptop when they graduate.
```
Comment (nickolaswilson-wf) [inline src/pages/index.md]: >socio-economic and social status

Shouldn't status be pluralized?

File: src/pages/index.md [hunk 4]
```diff
@@ -1,70 +1,26 @@
...
     - image: /img/coffee-gear.png
       text: >
-        We offer a small, but carefully curated selection of brewing gear and
-        tools for every taste and experience level. No matter if you roast your
-        own beans or just bought your first french press, you’ll find a gadget
-        to fall in love with in our shop.
+        We provide equipment directly where it is needed at the time that it makes the most impact. Professionals in tech come in to teach the high school level who in turn give back immediately by mentoring their younger peers. With small classes, we give the one-on-one attention that these students need. We hire high school students to give them their first opportunity in industry, giving them a foot in the door and a strong resume as soon as possible.
```
Comment (nickolaswilson-wf) [inline src/pages/index.md]: >high school
->
>high-school

Comment (nickolaswilson-wf) [reply src/pages/index.md]: Also, "giving them a foot in the door" reads weird. What about something like "allowing them to get their foot in the door"?

File: src/pages/index.md [hunk 5]
```diff
@@ -1,70 +1,26 @@
...
-    alt: A close-up of a paper filter filled with ground coffee
-    image: /img/products-grid3.jpg
-  image2:
-    alt: A green cup of a coffee on a wooden table
-    image: /img/products-grid2.jpg
-  image3:
-    alt: Coffee beans
-    image: /img/products-grid1.jpg
----
+        America has a systemic lack of diversity in STEM-related fields. Iowa has a workforce shortage for STEM-related jobs. Des Moines has a growing population of immigrant and refugee families with parents who have limited access to education. Their children need to be guided towards a future that can help lift their families out of poverty and set them up for ongoing success. Pi515 is taking on the inequity in STEM-related careers by directly addressing the pipeline. Each year, we embrace 100 students, particularly girls, from diverse backgrounds. We introduce them to teachers and companies with employees who are in STEM, providing role models that inspire them. Over 80% of our students are the first in their families to finish high school and go on to college
```
Comment (nickolaswilson-wf) [inline src/pages/index.md]: Consider joining the first two sentences with an "and".

**Review by nickolaswilson-wf — CHANGES_REQUESTED** [review]

File: src/pages/index.md (hunk 1 above)
Comment (stefankraus-wf) [reply src/pages/index.md]: I was going to give them emphasis, but nah

File: src/pages/index.md (hunk 4 above)
Comment (stefankraus-wf) [reply src/pages/index.md]: I think High School is correct, non-hyphenated. 
Changed the foot door wording

File: src/pages/index.md [hunk 6]
```diff
@@ -7,23 +7,23 @@ subheading: Education and opportunities for underserved youth
...
     - image: /img/coffee-gear.png
       text: >
-        We provide equipment directly where it is needed at the time that it makes the most impact. Professionals in tech come in to teach the high school level who in turn give back immediately by mentoring their younger peers. With small classes, we give the one-on-one attention that these students need. We hire high school students to give them their first opportunity in industry, giving them a foot in the door and a strong resume as soon as possible.
+        We provide equipment directly where it is needed at the time that it makes the most impact. Professionals in tech come in to teach the high school level who in turn give back immediately by mentoring their younger peers. With small classes, we give the one-on-one attention that these students need. We hire high school students to give them their first opportunity in industry, allowing them to get their foot in the door and a strong resume as soon as possible.
```
Comment (nickolaswilson-wf) [inline src/pages/index.md]: >allowing them to get their foot in the door and a strong resume as soon as possible.

>allowing them to get their foot in the door and build a strong resume as soon as possible

**Review by nickolaswilson-wf — CHANGES_REQUESTED** [review]

**Review by nickolaswilson-wf — APPROVED** [review]

**Pull request merged**